validator/testcases/javascript/predefinedentities.py

```diff
@@ -579,13 +579,30 @@
                       u"nsIDOMGeoGeolocation": {"dangerous":
                          "Use of the geolocation API by add-ons requires "
                          "prompting users for consent."},
-                      u"nsIX509CertDB": {"dangerous": {
-                          "description": "Access to the X509 certificate "
-                                         "database is potentially dangerous "
-                                         "and requires careful review by an "
-                                         "administrative reviewer.",
-                          "editors_only": True,
-                      }}}
+                      }
+
+CONTRACT_ENTITIES = {}
```

Comment:
I went back and forth on that one. I mainly just wanted the `CONTRACT_ENTITIES` and `INTERFACE_ENTITIES` lines to follow the same pattern. But I'm happy to change it to just a `dict()` on that line.